src/main/java/edu/tseidler/states/PlayerSetUpState.java

```diff
@@ -32,9 +32,9 @@ public GameState getNextState() {
         return new SetupSummaryState(this);
     }
 
-    private String getPlayerName(String number) {
+    private String getPlayerName(String playerNumber) {
```

Comment:
It still suggests that it is a number, not a string
maybe playerNumberText/String or something like this?